extension/scripts/features/travelfillmax/ttTravelFillMax.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 (async () => {
-	if (!isAbroad()) return;
+	if (!isAbroad() || !document.find(".travel-agency-market")) return;

```

Comment:
1. This is a PR for the people filter, not the travel fill max.
2. This check should not be done in here, will look where it should be.